src/plugins/auto_batch/auto_batch.cpp

```diff
@@ -27,10 +27,11 @@
 namespace AutoBatchPlugin {
 using namespace InferenceEngine;
 
-std::vector<std::string> supported_configKeys = {CONFIG_KEY(AUTO_BATCH_DEVICE_CONFIG),
-                                                 CONFIG_KEY(AUTO_BATCH_TIMEOUT),
-                                                 CONFIG_KEY(CACHE_DIR)};
-
+std::vector<std::string> supported_configKeys = {CONFIG_KEY(AUTO_BATCH_DEVICE_CONFIG)};
+std::vector<std::string> internal_supported_configKeys = {CONFIG_KEY(AUTO_BATCH_DEVICE_CONFIG),
+                                                          CONFIG_KEY(AUTO_BATCH_DEVICE_CONFIG),
```

Comment:
AUTO_BATCH_DEVICE_CONFIG is mentioned two times

Reply:
It is a duplication.